lib/diagnostic.js

```diff
@@ -5,22 +5,46 @@
 // and returns the minimum from that set
 // assume that all the argument are numbers
 const min = function min() {
+  let result = arguments[0];
 
+  for (let i = 1; i < arguments.length; i++) {
+    if (arguments[i] < result) {
+      result = arguments[i];
+    }
+  }
+
+  return result;
 };
 
 // Question 2
 // Write a function that takes zero or more arguments
 // and returns the sum of that set
 // assume that all the argument are numbers
 const sum = function sum() {
+  let result = 0;
+
+  for (let i = 0; i < arguments.length; i++) {
+    if (arguments) {
+      result += arguments[i];
+    } else {
+      result = undefined;
+    }
+  }
 
+  return result;
```

Comment:
The `if` statement isn't correct since it doesn't pass the test if there are no arguments.